src/parser.cpp

```diff
@@ -824,6 +824,7 @@ void Parser::ParserImpl::loadConnection(const ModelPtr &model, const XmlNodePtr
             std::string variable1Name;
             std::string variable2Name;
             XmlAttributePtr childAttribute = childNode->firstAttribute();
+            mappingId = "";
```

Comment:
Might want to replace with `mappingId.clear()`.

Reply:
052f1676